src/crudeditor-lib/middleware/notifications.js

```diff
@@ -116,10 +117,19 @@ const eventsMiddleware = /* istanbul ignore next */ ({ i18n, modelDefinition })
         type: 'error',
         timeOut: ERROR_NOTIFICATION_TIMEOUT,
         message: (Array.isArray(action.payload) ? action.payload : [action.payload]).
-          filter(err => err && typeof err === 'object' && err.message).
-          map(({ message }) => message).
+          filter(err => err && typeof err === 'object').
+          map(({ id, message, args }) => exists(id) ?
```

Comment:
`id` instead of `exists(id)` is more clear

Reply:
What if `id` = 0?